app/controllers/events_controller.rb

```diff
@@ -36,7 +36,7 @@ def redirect_unless_admin
   end
 
   def event_params
-    params.require(:event).permit(:start_time, :duration, :address, :capacity)
+    params.require(:event).permit(:start_time, :duration, :address, :capacity, :price_cents)
```

Comment:
I see here you've used :price_cents. Should we keep it consistent?

Reply:
The idea is that I don't know whether Chloe will create events that are integers or floats so to make it safe the form is only accepting prices in cents to avoid incorrect rounding

In the seed it doesn't matter which one we use bacause Chloé or another user will never touch that